docs/technical_specs/model_libraries/generic/unit_models/skeleton_unit.rst

```diff
@@ -0,0 +1,67 @@
+Skeleton Unit Model
+===============================
+
+The IDAES Skeleton Unit model represents a bare bones unit (hence, the name skeleton) that lets the user define a unit model with a custom set
+of variables and constraints that best suits their needs but that can be connected to other models from the unit model library. This unit can be used
+to either define a surrogate for a complex unit model or to define a first-principles based model that does not use IDAES
+ControlVolumes or property packages. To facilitate connection to other models in the unit model library, the user needs to add ports and populate with variables that match
+the port members of the models this will be connected to. To enable this, a special method is provided to enable adding ports with a specified set of variables.
+
+The two methods available for this unit are:
+
+1. ``add_ports``: Allows the user to define the inlet and outlet ports for the model and populate the port with a set of variables.
+2. ``initialize``: A default initialize method that will attempt to solve the model as defined or that can be substituted with a custom callback function provided by the user.
+
+Degrees of Freedom
+------------------
+
+There are no variables or constraints as this is a bare minimum skeleton that will be populated with variables and constraints
```

Comment:
The following three sections aren't really applicable in this case (as everything depends on the user). It would probably be better to give an overview of how to use the skeleton model, and what the user needs to keep in mind.

Some examples:

* Port members need to be indexed by time. That leads into how to access the time domain (I will add a PR for making `time` a property of the flowsheet).
* How to use the methods to add ports and how to define the members of the port.
* A reminder that Ports must be consistent, thus users need to match their Port definitions to the flowsheet they intend to use it in.

Reply:
Two other things that would be worth mentioning:

* unit consistency (if the user wishes to assert unit consistency across the model, then the Skeleton will need to be unit consistent)
* Brief mention of how this could interact with the scaling tools (for now, just say the user would need to do this manually).

Reply:
I plan to create an example in examples-pse